src/components/TeacherComponents/helpers/downloadAndShareHelpers/fetchHubspotContactDetails.ts

```diff
@@ -21,16 +24,24 @@ export async function fetchHubspotContactDetails(email: string) {
       throw new Error("Failed to fetch contact details");
     }
 
+    if (response.status === 204) {
+      return null;
+    }
+
     const result = contactDetails.parse(await response.json());
 
     return result;
   } catch (err) {
-    if (err instanceof OakError) {
-      throw err;
-    }
-    throw new OakError({
-      code: "hubspot/unknown",
-      originalError: err,
-    });
+    const error =
+      err instanceof OakError
+        ? err
+        : new OakError({
+            code: "hubspot/unknown",
+            originalError: err,
+          });
+
+    reportError(error);
```

Comment:
We weren't properly reporting this error which made it unhelpful in Bugsnag. This should now populate the `originalError` in Bugsnag metadata which will make it easier to trace in future.